sm/enclave.c

```diff
@@ -14,6 +14,10 @@
 #define ENCL_TIME_SLICE 100000
 
 struct enclave enclaves[ENCL_MAX];
+
+//Enclave mailbox metadata
+struct mailbox enclave_mailbox[ENCL_MAX]; 
```

Comment:
We can/should probably put the mailbox struct inside the enclave struct for simplicity.